src/resources/files.ts

```diff
@@ -134,6 +164,9 @@ function generateUploadEndpoint(
         data: fileContent,
       });
     }
+    if (waitUntilAcknowledged) {
+      await waitUntilFileIsUploaded(file.id, axiosInstance, resourcePath);
+    }
```

Comment:
should't we return a newer information here? at least `uploaded` is true now

Reply:
Good catch! Yes. I will fix